### ISSUE TITLE
Not able to run pmdarima from shell script

### ISSUE THREAD

**Issue by Kalyan2823** [alkaline-ml/pmdarima#430]
I am able to import pmdarima in anaconda environment. it is sucessful.

I am just trying to run a simple import command and printing the verison

```python
import pmdarima as pmd
print(pmd.__version__)
```

when i execute this using python run command (`python arima.py`), am getting the result :

```
1.8.0
```

But when i try to run this `python arima.py` command from a shell script : 

```bash
# !/bin/sh
python arima.py
```

I am getting error :

```
Traceback (most recent call last):
  File "arima.py", line 1, in <module>
    import pmdarima as pmd
ModuleNotFoundError: No module named 'pmdarima'
```

**Comment by aaronreidsmith** [comment]
It's very possible `/bin/sh` is picking up a different version of python than your default terminal, and the version it is using likely doesn't have `pmdarima` installed.

When you load up a terminal, it looks in several places to load some default settings. For example, if you are on mac, that can include `~/.bash_profile` and `~/.bashrc` (if you use `bash` as your default shell), or `~/.zsh_profile` and `~/.zshrc` (if you use `zsh` as your default shell). Let's look at an example:

```
aaron.smith ~ $ which python
python: aliased to /Library/Frameworks/Python.framework/Versions/3.9/bin/python3.9
aaron.smith ~ $ cat ~/test.sh
#!/bin/sh

which python
aaron.smith ~ $ ./test.sh
/usr/bin/python
```

As you can see, I must have something going on in my `~/.zshrc` file (since I use `zsh`) that is making `/bin/zsh` point to `/Library/Frameworks/Python.framework/Versions/3.9/bin/python3.9` when I call `python`. This is because I have this line in my `~/.zshrc` file:

```
alias python='/Library/Frameworks/Python.framework/Versions/3.9/bin/python3.9'
```

However, `/bin/sh` is _unaware_ that I have this, so it defaults to the system `python` in `/usr/bin/python`. I suspect you are having the same issues. Let's look at one more example:

```
aaron.smith ~ $ python -c "import pmdarima; print(pmdarima.__version__)"
1.8.2
aaron.smith ~ $ cat test.sh
#!/bin/sh

python -c "import pmdarima; print(pmdarima.__version__)"
aaron.smith ~ $ ./test.sh 
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: No module named pmdarima
```

Two things are apparent here:

1. From the terminal (`zsh`) we are able to import pmdarima and print its version
    - Note it is `pmdarima.__version__` and not `pmdarima.version`
2. We edited `test.sh` to look _the same_ as the terminal version and we get the same error as you!

This is because `/bin/sh` which is being specified in `test.sh` is using a different version of Python _where `pmdarima` is not installed_.

Let's change `test.sh` to point to the same version of Python as our terminal:

```
aaron.smith ~ $ which python
python: aliased to /Library/Frameworks/Python.framework/Versions/3.9/bin/python3.9
aaron.smith ~ $ cat test.sh 
#!/bin/sh

/Library/Frameworks/Python.framework/Versions/3.9/bin/python3.9 -c "import pmdarima; print(pmdarima.__version__)"
aaron.smith ~ $ ./test.sh 
1.8.2
```

And just like that, it works! 

This is a good reminder that we need to check that our runtime environment (`test.sh`) is the same as our test environment (the terminal).

Let me know if that solves your problem!

**TL;DR: Run `which python` and use the output of that command instead of `python` in your shell script. Also, use `pmd.__version__` instead of `pmd.version`**

**Comment by Kalyan2823** [comment]
Thanks a Ton aaron Smith for detailed explanation with examples. It worked and am able to import the pmdarima libraries successfully. 

Kudos to you 👍
